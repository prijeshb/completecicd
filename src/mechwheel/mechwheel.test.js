import React from "react";
import { Mechwheel } from "..";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Mechwheel />).toJSON();
  expect(tree).toMatchSnapshot();
});
