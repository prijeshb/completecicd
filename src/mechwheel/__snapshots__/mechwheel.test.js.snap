// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "color": "red",
    }
  }
>
  <h1>
    HEllo from mechwheel!!!
  </h1>
</div>
`;
